sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -52,6 +53,15 @@ trait FunctionRegistry {
   /** Drop a function and return whether the function existed. */
   def dropFunction(name: String): Boolean
 
+  /* Return the FunctionBuilder and ExpressionInfo for the specified function name and classname. */
+  def makeFunctionBuilderAndInfo(
+    name: String,
+    functionClassName: String): (ExpressionInfo, FunctionBuilder) = {
+    val clazz = Utils.getContextOrSparkClassLoader.loadClass(functionClassName)
+    val (_, (info, builder)) =
+      FunctionRegistry.expression(name, clazz.asInstanceOf[Class[Expression]])
```

Comment:
Instead I would just throw unsupported operation exception for now (e.g. what I do in #12051). This was never supported in the first place so we don't have to implement it now.